lib/src/metta/interpreter.rs

```diff
@@ -529,8 +529,8 @@ fn execute_op<'a, T: SpaceRef<'a>>(context: InterpreterContextRef<'a, T>, input:
             let mut expr = expr.clone();
             let op = expr.children().get(0).cloned();
             if let Some(Atom::Grounded(op)) = op {
-                let mut args = expr.children_mut().drain(1..).collect();
-                match op.execute(&mut args) {
+                let args: Vec<Atom> = expr.children_mut().drain(1..).collect();
+                match op.execute(&args[..]) {
```

Comment:
In this PR we can eliminate copying in the code above:
```rust
            let op = expr.children().get(0);
            if let Some(Atom::Grounded(op)) = op {
                let args = expr.children();
                match op.execute(&args[1..]) {
```

Reply:
I adopted your suggestion in 4cc22d6.